server/routes/wishCards.js

```diff
@@ -417,30 +599,30 @@ router.get(
     let itemChoices;
```

Comment:
Shouldn't this be earlier, in line 475, if you are replacing the if statements with switch?

Reply:
yeah I think soemthing got fucked up during merging, removed it